db/controllers/articles.controller.js

```diff
@@ -1,41 +1,58 @@
 const {
-    selectArticleById, selectArticles, selectCommentById
-
-  } = require("../models/articles.model.js");
+  selectArticleById,
+  selectArticles,
+  selectCommentById,
+  addComment
+} = require("../models/articles.model.js");
 
 exports.getArticleById = (req, res, next) => {
-    const article_id = req.params.article_id
-
- selectArticleById(article_id).then((result) =>{
-        res.status(200).send(result)
-    }).catch((err)=>{
-        console.log(err);
-        next(err);
-      });
-    };
+  const article_id = req.params.article_id;
+
+  selectArticleById(article_id)
+    .then((result) => {
+      res.status(200).send(result);
+    })
+    .catch((err) => {
+      console.log(err);
+      next(err);
+    });
+};
 
 exports.getArticles = (req, res, next) => {
-
-        selectArticles().then((articles) => {
-          console.log({articles});
-          res.status(200).send({ articles });
-        }).catch((err)=>{
-            console.log(err);
-            next(err);
-          });
-    
-      };
-
-      exports.getArticleComments = (req, res, next) => {
-
-        const article_id = req.params.article_id
-
-        selectCommentById(article_id).then((result) =>{
-          console.log(result, "result");
-          res.status(200).send(result)
-      }).catch((err)=>{
-          console.log(err);
-          next(err);
-        });
-    
-      };
\ No newline at end of file
+  selectArticles()
+    .then((articles) => {
+      console.log({ articles });
+      res.status(200).send({ articles });
+    })
+    .catch((err) => {
+      console.log(err);
+      next(err);
+    });
+};
+
+exports.getArticleComments = (req, res, next) => {
+  const article_id = req.params.article_id;
+
+  selectCommentById(article_id)
+    .then((result) => {
+      console.log(result, "result");
+      res.status(200).send(result);
+    })
+    .catch((err) => {
+      console.log(err);
+      next(err);
+    });
+};
+
+exports.postArticleComment = (req, res, next) => {
```

Comment:
As this is the articles controllers file but this is actually ultimately dealing with comments I think a new file would be appropriate. Think about the data that is being sent and into which table it is being inserted. Same with the model